pkgs/development/compilers/solc/default.nix

```
@@ -56,14 +56,9 @@ stdenv.mkDerivation rec {
     pushd ..
     # IPC tests need aleth avaliable, so we disable it
     sed -i "s/IPC_ENABLED=true/IPC_ENABLED=false\nIPC_FLAGS=\"--no-ipc\"/" ./scripts/tests.sh
-    for i in ./scripts/*.sh; do
-      patchShebangs "$i"
-    done
-    for i in ./scripts/*.py; do
-      patchShebangs "$i"
-    done
-    for i in ./test/*.sh; do
-      patchShebangs "$i"
+    for i in ./scripts/*.sh ./scripts/*.py ./test/*.sh; do
+      substituteInPlace "$i" --replace '#!/usr/bin/env' '#!${coreutils}/bin/env' \
+                             --replace '#! /usr/bin/env' '#!${coreutils}/bin/env' ;
```

this feels like a mistake, perhaps fix it upstream
```suggestion
                             --replace '#! /usr/bin/env' '#!${coreutils}/bin/env' ;
```

---

It would be nice to fix the aesthetic discrepancy upstream, but until that's done, this code works. I could also try again with just `patchShebangs "$i"`, since the actual big change that broke the build was the transition from python2 to python3, and this was twiddling I did before I understood the issue.

---

OK, it works when I restore the previous definition.